skopt/tests/test_space.py

```diff
@@ -278,7 +278,8 @@ def test_space_api():
     samples_transformed = space.transform(samples)
     assert_equal(samples_transformed.shape[0], len(samples))
     assert_equal(samples_transformed.shape[1], 1 + 1 + 3 + 1 + 1)
-    assert_array_equal(samples, space.inverse_transform(samples_transformed))
+    assert_array_almost_equal(samples,
+                              space.inverse_transform(samples_transformed))
```

Comment:
other round-trip'ing tests also use `almost_equal` which is the way to compare things that might contain floating point errors.